Bottom: 7a32a0378394a9e58f0d2f3428278786a58a02cd
Top:    ba829d5a57a092bf755f1c54e83f7732a4edda0b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 22:19:35 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 1d5773b..5541056 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -989,7 +989,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
                 ghid_flip_y ? port->view_y0 - PCB->MaxHeight :
                              -port->view_y0, 0);
   DrawAttached ();
-  DrawMark (TRUE);
+  DrawMark ();
   hidgl_flush_triangles (&buffer);
   glPopMatrix ();
